meeting_guide/migrations/0016_alter_group_gso_number_alter_meeting_area_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.5 on 2023-10-02 20:31
+
+from django.db import migrations, models
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("meeting_guide", "0015_auto_20211002_1711"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="group",
+            name="gso_number",
+            field=models.CharField(
+                blank=True,
+                help_text="General Service Office (GSO) number, if applicable.",
+                max_length=10,
+                null=True,
+                verbose_name="GSO Number",
```

Comment:
Had no idea what a GSO was. Added help text and a verbose name to add clarification.